crates/anvil/src/eth/backend/executor.rs

```diff
@@ -208,6 +208,12 @@ impl<'a, DB: Db + ?Sized, Validator: TransactionValidator> TransactionExecutor<'
     }
 
     fn env_for(&self, tx: &PendingTransaction) -> EnvWithHandlerCfg {
+        let mut tx_env = tx.to_revm_tx_env();
```

Comment:
we are still passing `tx.to_revm_tx_env()` into `new_with_cfg_env` instead of `tx_env` var

Reply:
gotcha, mind taking over this?

Reply:
yep sure